_codux/boards/ui-kits/ui-kit-buttons.board.tsx

```diff
@@ -0,0 +1,110 @@
+import { createBoard, Variant } from '@wixc3/react-board';
+import styles from '../../../src/styles/ui-kit-buttons.module.scss';
+import commonStyles from '~/styles/common-styles.module.scss';
+import classNames from 'classnames';
+import facebookIcon from '~/assets/svg/facebook.svg';
+import twitterxIcon from '~/assets/svg/twitterx.svg';
+import discordIcon from '~/assets/svg/discord.svg';
+import youtubeIcon from '~/assets/svg/youtube.svg';
+import mediumIcon from '~/assets/svg/medium.svg';
+import githubIcon from '~/assets/svg/github.svg';
+import { NavLink } from '@remix-run/react';
+import { MemoryRouter } from 'react-router-dom';
+
+export default createBoard({
+    name: 'UI Kit - Buttons',
+    Board: () => (
+        <MemoryRouter>
+            <div className={styles.container}>
+                <div>
+                    <span className={styles.uikit}>UI Kit</span>
+                    <span className={styles.coreComponents}> | Core Components</span>
+                    <hr className={styles.hrSolid} />
+                    <h3 className={styles.sectionTitle}>Buttons</h3>
+                </div>
+
+                <h4 className={styles.sectionHeader}>THEMED</h4>
+
+                <div className={classNames(styles.buttonsContainer, styles.itemSpacing)}>
+                    <div>
+                        <Variant name="Primary Button">
+                            <button className={classNames(commonStyles.primaryButton)}>
+                                Primary
+                            </button>
+                        </Variant>
+                        <span className={styles.buttonLabel}>Primary</span>
+                    </div>
+
+                    <div>
+                        <Variant name="Secondary Button">
+                            <button className={classNames(commonStyles.secondaryButton)}>
+                                Secondary
+                            </button>
+                        </Variant>
+                        <span className={styles.buttonLabel}>Secondary</span>
+                    </div>
+                </div>
```

Comment:
Same, it would be better to change the 2 outer `div` to `section`.

There are 2 more `div`s below (`main`, `iconsContainer`) that you can change below.